### ISSUE TITLE
Error when initializing `KickstartCompleter` object

### ISSUE THREAD

**Issue by zuh0** [pykickstart/pykickstart#339]
Hello,

When launching `ksshell` I get a Python stack trace because of an `AttributeError`.

Expected behaviour:

Launching `ksshell`.

Observed behavior:

A stack trace:

```
Traceback (most recent call last):
  File "/home/zuh0/src/pykickstart/venv/bin/ksshell", line 240, in <module>
    main()
  File "/home/zuh0/src/pykickstart/venv/bin/ksshell", line 187, in main
    readline.set_completer(KickstartCompleter(kshandler, internalCommands).complete)
  File "/home/zuh0/src/pykickstart/venv/bin/ksshell", line 84, in __init__
    self._add_command(cStr, cObj)
  File "/home/zuh0/src/pykickstart/venv/bin/ksshell", line 98, in _add_command
    for opt in cObj.op.option_list:
AttributeError: 'KSOptionParser' object has no attribute 'option_list'
```

Steps to reproduce:

```
$ git clone https://github.com/pykickstart/pykickstart
$ cd pykickstart
$ python3 -m venv venv
$ . venv/bin/activate
$ python3 -m pip install -r requirements.txt
$ ksshell
```

Python version:  3.8.5
Linux distribution: Fedora 32

I have also checked in the official Fedora 32 repositories and the packaged version is also broken in the same way.

**Comment by bcl** [comment]
Thanks, I've filed a bugzilla bug for this - https://bugzilla.redhat.com/show_bug.cgi?id=1878242

**Comment by bcl** [comment]
Fixed by PR #340
